terraform/environment/dns.tf

```diff
@@ -0,0 +1,29 @@
+variable "subdomains" {
+  type = list(string)
+  default = [
+    "nginz-https",
+    "nginz-ssl",
+    "webapp",
+    "assets",
+    "account",
+    "teams"
+  ]
+}
+
+variable "create_spf_record" {
+  type = bool
+  default = true
```

Comment:
suggestion: defaulting to `false` makes more sense, since it's mostly to facilitate the success of mail sending from the backend (`charts/demo-smtp`)

Reply:
Sure, this was the default before. I'll change it.